### ISSUE TITLE
Attempting to load xhook.js from unpkg.com results Blank Screen on startup.

### ISSUE THREAD

**Issue by wmltogether** [ciderapp/Cider-2#469]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

2.2.0

### Operating System Version

Windows 11 22H2

### Describe the Bug

Actually, this issue is not a bug, the unpkg.com is blocked on my environment.
When xhook.min.js fails to load, the screen goes blank at startup. 
Is it possible to embed all scripts with Tauri instead of loads from remote CDN?

<img width="809" alt="Snipaste_2023-12-15_10-10-07" src="https://github.com/ciderapp/Cider-2/assets/10459551/9d01028d-3e79-4c91-b0d9-557b22adfbdf">

### Steps to Reproduce

1. Block unpkg.com using firewall 
2. Restart App

**Comment by coredev-uk** [comment]
This issue has been closed to consolidate the issue tracker for Cider 2.3.1 and later issues. If you believe this issue is still relevant, please open a new issue.
